src/test/java/seedu/address/logic/parser/FindAppointmentCommandParserTest.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.logic.parser;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.FindAppointmentCommand;
+
+public class FindAppointmentCommandParserTest {
+
+    @Test
+    public void parse_noPrefix_failure() {
+        FindAppointmentCommandParser parser = new FindAppointmentCommandParser();;
```

Comment:
I see 2 semicolons in this line, I suggest please remove one.

Reply:
Surprising this passed checkstyle, thanks!